src/main/asciidoc/batch/file-ingest-sftp/local.adoc

```diff
@@ -0,0 +1,209 @@
+[[sftp-file-ingest-local]]
+==== Using the Local Server
+
+===== Additional Prerequisites
+
+* A running local Data Flow Server
+include::{docs_dir}/local-server.adoc[]
+
+* Running instance of link:http://kafka.apache.org/downloads.html[Kafka]
+* Either a remote or local host accepting SFTP connections.
+
+* A database tool such as link:https://dbeaver.jkiss.org/download/[DBeaver] to inspect the database contents
+
+NOTE: To simplify the dependencies and configuration in this example, we will use our local machine acting as an SFTP server.
+
+===== Building and Running the Demo
+
+. Build the demo JAR
++
+From the root of this project:
++
+```
+$ cd batch/file-ingest
+$ mvn clean package
+```
+NOTE: For convenience, you can skip this step.
+The jar is published to the  https://repo.spring.io/libs-snapshot-local/io/spring/cloud/dataflow/ingest/ingest/1.0.0.BUILD-SNAPSHOT/[Spring Maven repository]
++
+
+. Create the data directories
++
+Now we create a remote directory on the SFTP server and a local directory where the batch job expects to find files.
++
+NOTE: If you are using a remote SFTP server, create the remote directory on the SFTP server.
```

Comment:
Maybe clarify `/tmp/remote-files` is in local because we are running the SFTP server on localhost.

Reply:
Hmmm. The update doesn't show here. I made the change though.